### PR TITLE
Fix 404 link to the pyporting mailing list

### PR DESCRIPTION
**Pull request by rohitnishad613**
https://mail.python.org/mailman/listinfo/python-porting this link is broken and return 404 error. So i just removed it.
Also, this page is about proting code so i linked out to my own python converter tool. (Its not a paid tool)

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@rohitnishad613

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by rohitnishad613** [comment]
I signed PSF CA but added...
Is there anything wrong?

**Comment by Mariatta** [comment]
@rohitnishad613 You also need to create the b.p.o account, and then put in your GitHub username in the b.p.o profile.

**Comment by Mariatta** [comment]
So python-porting mailing list used to exist but it was retired a couple years ago.
See the thread about it here: https://mail.python.org/archives/list/python-dev@python.org/thread/6436V5ATEGKUGRQU5ONV27PUL2O4TMR4/

The archive is still accessible at  https://mail.python.org/pipermail/python-porting/
I would simply replace the link with this one, instead of recommending a third-party tool.

File: Doc/howto/pyporting.rst [hunk 1]
```diff
@@ -20,8 +20,8 @@ Porting Python 2 Code to Python 3
    came into existence, you can read Nick Coghlan's `Python 3 Q & A`_ or
    Brett Cannon's `Why Python 3 exists`_.
 
-   For help with porting, you can email the python-porting_ mailing list with
-   questions.
+   
+   For help with porting, you can use this python-converter-tool_.
```
Comment (Mariatta) [inline Doc/howto/pyporting.rst]:
```suggestion
   For help with porting, you can view the archived python-porting_ mailing list.
```

File: Doc/howto/pyporting.rst [hunk 2]
```diff
@@ -446,7 +446,7 @@ to make sure everything functions as expected in both versions of Python.
 
 .. _pytype: https://github.com/google/pytype
 .. _python-future: http://python-future.org/
-.. _python-porting: https://mail.python.org/mailman/listinfo/python-porting
+.. _python-converter-tool: https://360techexplorer.com/tools/python-2-to-3-converter/
```
Comment (Mariatta) [inline Doc/howto/pyporting.rst]:
```suggestion
.. _python-porting: https://mail.python.org/pipermail/python-porting/
```

**Review by Mariatta — CHANGES_REQUESTED** [review]
I prefer we link to the archived mailing list instead of linking to a third party tool.

**Comment by rohitnishad613** [comment]
I have made the requested changes; please review again.

Plus, I have a question I made a web tool "Python 2 to 3 converter" I think that tool is worth linking so, can I link it here?

**Pull request merged**